src/components/Footer/Footer.js

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable react/prop-types */
+// componenets/Footer.js
+import React from "react";
+
+import Link from "next/link";
+
+import Image from 'next/image';
+
+import StyledFooter from './FooterStyle.js';
+
+import {StyledFooterContainer, StyledLogo, StyledRow1, StyledRow2, StyledRow3, StyledLinks, StyledUl, StyledLi, StyledLiHeader} from './FooterStyle.js';
+
+  
+
+const Footer = props => (
+    <StyledFooter>
+        <StyledFooterContainer>
+            <StyledLogo>
+                <Image src='/TextLogo.png' alt='TextLogo' width='150' height='70' />
+            </StyledLogo>
+            <StyledRow1>
+                <StyledUl>
+                    <StyledLiHeader>FOR JOB SEEKERS</StyledLiHeader>
+                    <StyledLi>Browse Jobs</StyledLi>
+                    <StyledLi>Browse Contacts</StyledLi>
+                    <StyledLi>Skills</StyledLi>
+                    <StyledLi>Visualize Data</StyledLi>
+                </StyledUl>
```

Comment:
Rather than StyledUl and StyledLi components, I think it would be better to have the styling for this in StyledFooter like we discussed in our last meeting. This is because the styling for lists is all the same throughout the footer, so you don't necessarily need separate components and can just add styling like this: 

`
li {
/* styling here */
}
`

Reply:
Done!  Turns out I made that StyledLi tag but wasn't using it.  only StyledLiHeader has any different CSS.  Changed all StyledLi's to just li's